contrib/migration_script.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+"""
+This script is for migrating jobs to another namespace
+"""
+import argparse
+import subprocess
+import time
+
+import yaml
+from six.moves.urllib.parse import urljoin
+from six.moves.urllib.parse import urlparse
+
+from tron.commands import client
+
+
+class bcolors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Migrate jobs to new namespace'
+    )
+    parser.add_argument(
+        '--server',
+        required=True,
+        # default='http://tronplayground-uswest1adevc.dev.yelpcorp.com:8089',
+        help='specify the location of tron master'
+    )
+    parser.add_argument(
+        '--old-ns',
+        required=True,
+        # default='dsc-test-service',
+        help='Old namespace'
+    )
+    parser.add_argument(
+        '--new-ns',
+        required=True,
+        # default='test_namespace',
+        help='New namespace'
+    )
+    parser.add_argument(
+        'source',
+        help='source file to get list of jobs'
+    )
+    args = parser.parse_args()
+    return args
+
+
+def check_job_if_running(jobs_status, job_name):
+    for job_status in jobs_status:
+        if job_status['name'] == job_name:
+            status = job_status['status']
+            if status == 'running':
+                print(bcolors.FAIL + 'job {} is still running, can not migrate'.format(job_name) + bcolors.ENDC)
+                return False
+            else:
+                print(bcolors.OKGREEN + 'job {} is not running, can migrate'.format(job_name) + bcolors.ENDC)
+                return True
+
+    print(bcolors.FAIL + 'Can not find the job {}'.format(job_name) + bcolors.ENDC)
+    return False
+
+
+def command_jobs(command, jobs, args):
+    data = {'command': command}
+    command_flag = True
+    for job in jobs:
+        if command == 'enable':
+            job_name = args.new_ns + '.' + job['name']
+        else:
+            job_name = args.old_ns + '.' + job['name']
+
+        if command == 'move':
+            data = {'command': command, 'old_name': args.old_ns + '.' + job['name'], 'new_name': args.new_ns + '.' + job['name']}
+            uri = urljoin(args.server, 'api/jobs')
+        else:
+            data = {'command': command}
+            uri = urljoin(args.server, 'api/jobs/' + job_name)
+
+        response = client.request(uri, data=data)
+        if response.error:
+            print(bcolors.FAIL + 'Failed to {} {}'.format(command, job_name) + bcolors.ENDC)
+            command_flag = False
+        else:
+            print(bcolors.OKGREEN + 'Succeed to {} {}'.format(command, job_name) + bcolors.ENDC)
+    return command_flag
+
+
+def ssh_command(hostname, command):
```

Comment:
Printing out the command that is running will be helpful for debugging and recovery.

This script is for us, it is OK to be verbose